d2core/d2asset/composite.go

```diff
@@ -54,7 +55,8 @@ func (c *Composite) Render(target d2interface.Surface) error {
 		return nil
 	}
 
-	for _, layerIndex := range c.mode.drawOrder[c.mode.frameIndex] {
+	for idx := range c.mode.drawOrder[c.mode.frameIndex] {
+		layerIndex := c.mode.drawOrder[c.mode.frameIndex][idx]
```

Comment:
i think this is making a local copy of `c.mode.drawOrder[c.mode.frameIndex][idx]`